### DISCUSSION TITLE
Docker Image

### DISCUSSION THREAD

**Discussion by akrea** [Rem0o/FanControl.Releases#676] [Ideas]
Hi
FANTASTC piece of work! I would love to use this on my server too. I know about the post asking for a Linux version. All services on my server run on Docker. Any chance that there will be a Docker image?
I can point the direction for setting one up. But my knowledge is not sufficient to build one myself.
cheers! How can I donate?

**Comment by Rem0o** [comment]
Hardware control through virtualized containers? I don't think that would even work in principle, even less so in practice.

**Reply by akrea** [reply]
I did not try this one
https://github.com/pilotak/docker-rpi-fan
but it seems possible...
